applications/OptimizationApplication/tests/test_container_expression.py

```diff
@@ -0,0 +1,364 @@
+
+from abc import ABC
+from abc import abstractmethod
```

Comment:
FYI you can do this too.
Don't accept it tho, it's not worth rerunning the CI for this.

```suggestion
from abc import ABC, abstractmethod
```